decidim-assemblies/app/serializers/decidim/assemblies/assembly_serializer.rb

```diff
@@ -9,6 +9,7 @@ def serialize
         super.merge(
           {
             categories: serialize_categories,
+            taxonomies: serialize_taxonomies(resource),
```

Comment:
See my previous comment. 
```suggestion
            taxonomies: ,
```